frame/contracts/src/weights.rs

```diff
@@ -156,32 +156,32 @@ pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: Contracts DeletionQueue (r:1 w:0)
 	fn on_initialize() -> Weight {
-		(1_636_000 as Weight)
+		(1_601_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 	}
 	// Storage: Skipped Metadata (r:0 w:0)
 	fn on_initialize_per_trie_key(k: u32, ) -> Weight {
-		(7_840_000 as Weight)
+		(10_543_000 as Weight)
 			// Standard Error: 0
-			.saturating_add((752_000 as Weight).saturating_mul(k as Weight))
+			.saturating_add((739_000 as Weight).saturating_mul(k as Weight))
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 			.saturating_add(T::DbWeight::get().writes((1 as Weight).saturating_mul(k as Weight)))
 	}
 	// Storage: Contracts DeletionQueue (r:1 w:0)
 	fn on_initialize_per_queue_item(q: u32, ) -> Weight {
-		(31_915_000 as Weight)
-			// Standard Error: 1_000
-			.saturating_add((98_000 as Weight).saturating_mul(q as Weight))
+		(0 as Weight)
+			// Standard Error: 3_000
+			.saturating_add((2_285_000 as Weight).saturating_mul(q as Weight))
```

Comment:
This increase is expected as the buggy code before wasn't doing doing much per queue item.